### ISSUE TITLE
SequelizeUniqueConstraintError when releasing to prod

### ISSUE THREAD

**Issue by ccanash** [w3c/aria-at-app#1235]
When re-applying the NVDA Bot to Navigation Menu on production, there were SequelizeUniqueConstraintError console errors and it failed to assign. They pointed to CollectionJob.id. It was trying to create a new row with id 19 when the latest auto-incremented id in the table was 33 (with no missing rows between 19 and 33).

**Comment by ccanash** [comment]
Closing  as we couldn't reproduce it.
